bftcosi/bftcosi.go

```diff
@@ -290,7 +290,7 @@ func (bft *ProtocolBFTCoSi) handleCommitment(msgs []commitChan) error {
 			if len(bft.tempPrepareCommit) < len(bft.Children()) {
 				continue
 			}
-			commitment = bft.prepare.Commit(nil, bft.tempPrepareCommit)
+			commitment = bft.prepare.Commit(bft.Suite().RandomStream(), bft.tempPrepareCommit)
```

Comment:
Question: why is this not just `bft.prepare.Commit(bft.Suite(), ...)`?

Same for `Commit` and `CreateCommitment` below.